src/seedu/addressbook/commands/Command.java

```diff
@@ -16,6 +16,15 @@
     protected List<? extends ReadOnlyPerson> relevantPersons;
     private int targetIndex = -1;
 
+    /**
+     * Use this to decide if the addressbook data needs to be saved to disk
```

Comment:
Header comment does not conform with coding style. Should be on the form of 'Returns true if....` or something along those lines.